tests/data/tag_linux.py

```diff
@@ -32,84 +32,114 @@ class LinuxTaggingFileTest(test_lib.TaggingFileTestCase):
 
   _TEST_TIMESTAMP = timelib.Timestamp.CopyFromString('2020-04-04 14:56:39')
 
+  def _CheckTaggingRule(
+      self, event_data_class, attribute_values_per_name, expected_rule_names):
+    """Tests a tagging rule.
+
+    Args:
+      event_data_class (type): class of the even data object to use in tests.
+      attribute_values_per_name (dict[str, list[str]): attribute values per
```

Comment:
Please clarify the docstring here - I don't understand what's supposed to be in this dict.

Reply:
updated